GetStream/stream-chat-android
# Merge channel invites into one section

### 🧪 Testing

1. Go to documentation folder
2. Run `yarn run start`
3. Check out Client -> Channels Invites section

### ☑️ Checklist
